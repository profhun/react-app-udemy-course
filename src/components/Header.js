import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = (props) => (
    <header>
        <h1>Kiad√°sok</h1>
        <div>
            <NavLink to="/" activeClassName="is-active" exact={true}>~</NavLink>
            <NavLink to="/add" activeClassName="is-active">+</NavLink>
            <NavLink to="/help" activeClassName="is-active">?</NavLink>
        </div>
    </header>
);

export default Header;