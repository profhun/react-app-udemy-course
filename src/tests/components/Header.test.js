import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

test('render/component Header', () => {

    const wrapper = shallow(<Header />);
    expect(wrapper.find('h1').text()).toBe("Kiad√°sok");
    expect(wrapper).toMatchSnapshot();
});