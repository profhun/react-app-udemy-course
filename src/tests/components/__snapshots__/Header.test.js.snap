// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render/component Header 1`] = `
<header>
  <h1>
    Kiad√°sok
  </h1>
  <div>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      exact={true}
      to="/"
    >
      ~
    </NavLink>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      to="/add"
    >
      +
    </NavLink>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      to="/help"
    >
      ?
    </NavLink>
  </div>
</header>
`;
